docs/src/development/planning_meetings.md

```diff
@@ -1,20 +1,23 @@
-# Roadmap
+# Planning Meetings
 
-A roadmap with short-, mid- and long-term planning will be available soon. For now, there is only a backlog.
+We maintain a public board for planning development tasks in metal-stack on GitHub: https://github.com/orgs/metal-stack/projects/34.
```

Comment:
Info: not public yet as it's not finished and cleaned up yet.
This comment just acts as a TODO.